x-pack/solutions/search/packages/kbn-search-index-documents/components/document_list.tsx

```diff
@@ -70,15 +70,6 @@ export const DocumentList: React.FC<DocumentListProps> = ({
   const pageCount = meta?.pageSize ? Math.ceil(meta.totalItemCount / meta?.pageSize) : 0;
   return (
     <>
-      <EuiPagination
```

Comment:
It might be worth adding a prop to control this instead of just removing it completely?

Something like `paginationDisplay={Top|Bottom|Both}`, but also not sure if we're losing much by removing this in the other components that use the `DocumentList`.

@julianrosado @mdefazio opinions?

Reply:
Yep! we could do that. Added this  based on my discussion with @mdefazio and @julianrosado that we are okay to apply the changes in this PR to all the  components that use [kbn-search-index-documents](https://github.com/elastic/kibana/tree/main/x-pack/solutions/search/packages/kbn-search-index-documents).  
But currently only this change(removing top pagination) is the only change affecting all consuming components, which is `connector details -> documents tab`